examples/asset/utilization/AssetUtilization.cs

```diff
@@ -0,0 +1,46 @@
+﻿using NationalInstruments.SystemLink.Clients.AssetManagement;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.Asset.Utilization
+{
+    /// <summary>
+    /// Example for the SystemLink Asset Management Client API that
+    /// tracks utilization for assets present in the system.
+    /// 
+    /// Note: As a prerequisite for this example, ensure SystemLink Client is installed.
+    /// </summary>
+    class AssetUtilization
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * The Asset Utilization Store enables asset utilization tracking. 
+             * It is created using a factory.
+             */
+            var utilizationStoreFactory = new AssetUtilizationStoreFactory();
+            var assetUtilizationStore = utilizationStoreFactory.CreateAssetUtilizationStore();
+
+            /*
+             * Configuring the API to send heartbeats for ongoing sessions at predefined intervals.
+             * By default, heartbeats are automatically sent every 5 minutes.
+             */
+            var utilizationConfiguration = StartUtilizationConfiguration.CreateDefault();
+
+            /*
+             * Start a utilization session for all assets in the system.
+             * For starting a utilization session, the following information is nedeed:
+             * - The utilization configuration
+             * - [Optional] asset name(s): a single asset name or a collection of asset names. if this parameter is omitted, all assets are tracked.
+             * - Utilization category: The reason for using the asset(s), such as for tests, configuration, maintenance, or custom purposes.
+             * - User name: The name of the operator using the asset.
+             * - Application name: The program which is using the asset(s).
```

Comment:
Just curious, do we automatically fill in this information based on the running EXE if it's omitted?

Reply:
Only in the LabVIEW API. In the Client App, we explicitly pass in the localized string.